exploration/0003-formatted-parts.md

```diff
@@ -0,0 +1,205 @@
+# Formatted Parts
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@eemeli</dd>
+		<dt>First proposed</dt>
+		<dd>2023-08-29</dd>
+		<dt>Pull Request</dt>
+		<dd>#000</dd>
+	</dl>
+</details>
+
+## Objective
+
+Messages often include placeholders that,
+when formatted, contain internal structure ("parts")
+that the caller might want access to
+for the purposes of styling, presentation, or manipulation.
+
+This proposal defines a formatted-parts target for MessageFormat 2.
+
+## Background
+
+Past examples have shown us that if we don't provide a formatter to parts,
+the string output will be re-parsed and re-processed by users.
```

Comment:
It might be useful to explain why it's undesirable for users to re-parse a string (or cite some other document that explains it).

Reply:
```suggestion
the string output will be re-parsed and re-processed by users.
Recent examples of web browsers needing to account for such user behaviour are available from
[June 2022](https://github.com/WebKit/WebKit/commit/1dc01f753d89a85ee19df8e8bd75f4aece80c594) and
[November 2022](https://bugs.chromium.org/p/v8/issues/detail?id=13494).
```